src/altinn-app-frontend/src/utils/formLayout.ts

```diff
@@ -468,3 +469,11 @@ export function isDatePickerComponent(
 ): component is IDatePickerProps & ILayoutComponent {
   return component.type.toLowerCase() === 'datepicker';
 }
+
+export function isCheckboxesComponent(
```

Comment:
Add some tests? 😇

Reply:
Yup, I can do that, but I prefer to do it in a later PR. There are some changes to these functions in the [hackaton/dynamics](https://github.com/Altinn/app-frontend-react/compare/main...hackaton/dynamics) branch which I'd like to port over, and I've thought about possibly changing up some of the typings. The component types are a prime example of a case where [discriminated unions](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#discriminated-unions) works wonders - so with a change in that direction, these functions could be removed altogether, as TypeScript would give us the same functionality out of the box. I'll add it to my list of sabbatical TODOs.. :wink: